packages/styling/src/styles/theme.ts

```diff
@@ -108,21 +108,30 @@ export function createTheme(theme: IPartialTheme, depComments: boolean = false):
     ...theme.semanticColors
   };
 
-  const typography = { ...DefaultTypography, ...theme.typography };
+  let typography = { ...DefaultTypography };
```

Comment:
In regards to the casting below, can't we just declare this ITypography and avoid the cast?

Reply:
Declaring this as ITypography results in a compiler error in lines 113-118 below. Since theme.typography has type IPartialTypography, theme.typography.families is possibly undefined, so there is an error in attempting to mix it in with typography.families.